src/main/java/database/Sample Data - Sample Item.csv

```diff
@@ -1,23 +1,23 @@
-SerialNumber, Name,Price,Quantity,Categories,ExpirationDate ,StorageLocation
-1,Ice Cream,9.99,5,,,
-2,Lettuce,3,22,,,
-3,Tomato,2,1,,,
-4,Cookie Dough,5,7777,,,
-5,Pizza,0.01,2,,,
-6,Candy,100,100,,,
-7,Burger,1200,2,,,
-8,Chicken,2,70,,,
-9,Fries,4,90,,,
-10,Garbage Bag,15,30,,,
-11,Apple,20,250,,,
-12,Strawberries,4.99,30,,,
-13,Almond Milk,5.99,30,,,
-14,Oat Milk,6.99,40,,,
-15,Goat Milk,2.99,60,,,
-16,Ground Beef,3.99,40,,,
-17,Carrot,1.99,70,,,
-18,Milk,2.99,60,,,
-19,Yogurt,3.99,45,,,
-20,Shoe sole,999.99,50,,,
-21,Cutting Board,10.99,60,,,
-22,Blueberries,4.99,0,,,
\ No newline at end of file
+SerialNumber, Name,Price,Quantity,Catagories,ExpirationDate ,StorageLocation
+1,Ice Cream,9.99,5,N/A,12/1/2022,Kansas
+2,Lettuce,3,22,N/A,12/2/2022,Kansas
+3,Tomato,2,1,N/A,12/3/2022,Kansas
+4,Cookie Dough,5,7777,N/A,12/4/2022,Kansas
+5,Pizza,0.01,2,N/A,12/5/2022,Kansas
+6,Candy,100,100,N/A,12/6/2022,Kansas
+7,Burger,1200,2,N/A,12/7/2022,Kansas
+8,Chicken,2,70,N/A,12/8/2022,Kansas
+9,Fries,4,90,N/A,12/9/2022,Kansas
+10,Garbage Bag,15,30,N/A,12/10/2022,Kansas
```

Comment:
Be careful about the data format you use. It should be corresponding to the example you write.